src/Workspaces/Core/Portable/Diagnostics/ShadowCopyAndAllowRedirectsAssemblyLoader.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// We'll only build this into our netstasndard2.0 build to target .NET Framework;
+// if you're on .NET Core then you should be using Assembly Load Contexts which removes
+// the need to use this type entirely.
+
+#if !NETCOREAPP
```

Comment:
Note, this is not included in our netcoreapp version, which is similar to what we do lower in the stack: the default one is implemented differently depending on target.

Reply:
This is now obsolete since we're not doing different behavior.